divi/src/coins.h

```diff
@@ -360,18 +367,19 @@ class CCoinsViewCache : public CCoinsViewBacked
 {
 protected:
     /* Whether this cache has an active modifier. */
-    bool hasModifier;
+    bool hasModifier = false;
 
     /**
      * Make mutable so that we can "fill the cache" even from Get-methods
      * declared as "const".
      */
-    mutable uint256 hashBlock;
+    mutable uint256 hashBlock = 0;
```

Comment:
Can we maybe leave the initialization in the member variables in the source?

Reply:
I personally prefer to initialise variables like this, as it avoids redundant code in multiple constructors.  But I understand if you prefer to keep the existing code / do it in that way, so I've reverted that part of the change.